cg/cli/set.py

```diff
@@ -170,13 +170,15 @@ def flowcell(context, flowcell_name, status):
 
 @set_cmd.command('microbial-order')
 @click.option('-a', '--application-tag', 'apptag', help='sets application tag.', type=str)
+@click.option('-t', '--ticket', 'ticket', help='sets ticket number.', type=str)
+@click.option('-n', '--name', 'name', help='sets name both in status-db and LIMS.', type=str)
 @click.argument('order_id')
 @click.argument('user_signature')
 @click.pass_context
-def microbial_order(context, apptag, order_id, user_signature):
+def microbial_order(context, apptag, ticket, name, order_id, user_signature):
```

Comment:
changed signature and no accompanying test? ;)

Reply:
just manual testing

Reply:
added tests